triaging.rst

```diff
@@ -35,17 +35,23 @@ Responsibilities include:
     - Good first issue
     - Other categorizations
 
-As triagers gain experience, they may have some intuition of when a PR should
-be closed. Triagers can recommend closing a PR, but the final decision must be
-made by a core developer. By having triagers and core developers work together,
+Although triagers have the power to close PRs, they should generally not do so
+without first consulting a core developer. By having triagers and core developers work together,
 the author receives a careful consideration of their PR. This encourages future
 contributions, regardless of whether their PR is accepted or closed.
 
-Triagers can make use of the ``invalid`` and ``stale`` labels to suggest that a
+Nonetheless, triagers should feel free to close a PR if they judge that the
+chance of the PR being merged is exceedingly low. This can include PRs that
+make solely cosmetic changes to a module, PRs that propose changes to
+deprecated modules, and/or PRs that self-evidently lie far below the quality
```

Comment:
also:  PRs which are no longer necessary because the issue had been fixed or made irrelevant by other PRs merged since this one was created.

Reply:
Non-coredev triagers can close issues in certain circumstances (which is a separate discussion) and when then do, they should close any associated PR.  (Same generally true for coredevs.)  You should say at least that much and remove any existing blanket permission denial.  Closing PRs for legitimate open issues is trickier.  I agree with sticking with the most obvious cases and not trying to be complete.  If in doubt, asking for a second opinion is not a bad thing.